dbms/src/Core/Block.h

```diff
@@ -52,6 +53,10 @@ class Block
     // Only used for calculating MVCC-bitmap-filter.
     ColumnPtr segment_row_id_col;
 
+    // `region_id` is use to trace which region this block belongs to when writing.
+    // Only use for logging and debugging.
+    RegionID region_id = 0;
```

Comment:
I think it is not a good idea to add a `RegionID` to `Block` that is being widely used everywhere in the code.

Reply:
I agree with @JaySon-Huang , Region and Segments are not always strictly mapped